### ISSUE TITLE
Ares head + 45 piece + imp gold pieces = phorcys salade

### ISSUE THREAD

**Issue by Strafeous** [LegionDark/Issues#164]
**Date & Time**: 7/15/2015   9:11pm
**Frequency**: once
**FFXI Client Version**:30150619_2
**Character Name**: Strafeous
**Race**: Elvaan
**Main Job**: WAR
**Sub Job**: SAM
**Area & Coordinates**: Whitegate at the Salvage/nyzul gear mithra
**Party or Solo**: Solo
**NPC Name**: Mrohk Sahjuuli
**Monster Name**: -
**Multi-boxing? (multiple clients on same connection)**: Yes
**Steps To Reproduce The Issue**:
Only did it once, but traded Ares Flanchard, the 45 piece, and 5 imp gold to Mrohk and it gave me a phorcys salade :P

**Comment by Loki79** [comment]
@TeoTwawki  i think you wrote this one, can you take a look at it

**Comment by TeoTwawki** [comment]
@Loki79 wasn't me, probably an Omni script. Sounds like item ID incorrect on reward item. Will take a look if I get time.

**Comment by TeoTwawki** [comment]
@Loki79 looks and smells like an omni script. missing a safety checks like "does player have a free space?" :anguished:

**Comment by Loki79** [comment]
ill add this to my re-write list for some day when im lacking shit to do, or when here something else about it.

**Comment by whitemistita** [comment]
same happened to me, whitemist (in april)

**Comment by synafey** [comment]
Happende to me too in April (Synafey)

**Comment by Loki79** [comment]
ill re-write the script sometime in the next few days.

**Comment by TeoTwawki** [comment]
@Loki79 I'm pretty sure I fixed the error part that did that, but they'd need manual adjustment of their item. That NPC could still be improved with a rewrite though, I just quick fixed the exact prob that was reported and nothing else.

**Comment by whitemistita** [comment]
"Manual adjustment" means we have to tell a GM in game?

**Comment by whitemistita** [comment]
ok thanks Teo :)

**Comment by whitemistita** [comment]
still no news from GMs. I'll post here when it happens and the issue will be solved

**Comment by TeoTwawki** [comment]
@LegionDark/owners Who do we have with `@giveitem` powers? They shouldn't be waiting this long but none of us dev tier dudes have had time to be in game and this needs taken care of.

**Comment by PikaFox** [comment]
Me and Roamin are the only GMs without giveitem powers afaik.

**Comment by Kaoriasada** [comment]
@TeoTwawki @PikaFox This bug report was made in July and marked as a trial bugfix, I wasn't scrolling through months old bug reports to double check and make sure no one had commented on it.  I definitely didn't see their comments, or I would have gotten to it as soon as I had saw it.  I check the bug reports everyday and log in when it's something I have to do.  I simply didn't see these and don't have the ability/coding knowledge to fix trials myself.  

@whitemistita I'll get you in-game next time I see you.  @synafey did anyone ever get to you?

**Comment by TeoTwawki** [comment]
Yeah sorry..
 @Kaoriasada @PikaFox I didn't expect more people to have been hit by it before the fix I pushed through, and left the issue open because of the planned rewrite to this. Had had tagged Loki after fixing it to let him know, and all 3 of myself Loki and R3 all get messaged by github when I use that group tag. I never remember who has which in game powers.

There has been some lack of standard/communication about when players should "me to" an issue and when to just make a new one, which I guess we'll all need to get together on IRC about once everyone is back. 
My current opinion is if its gonna take GM assistance in game or if an existing issue is too cluttered to keep track of who we already got to, we should go ahead and have them make "duplicate" reports. Example: that ToAU KI thing..we've been accepting multiple issues for that.

I'd been discouraging me-to posts and saved duplicate tag for those that didn't require direct help in game before but other staff seemed to prefer those over duplicate issues so I had stopped asking people to open separate ones.

**Comment by TeoTwawki** [comment]
tl;dr 
devs effed up by not assigning someone when they weren't around, sorry

**Comment by whitemistita** [comment]
Syna and I will log at 20:30 gmt +1. Thanks guys

**Comment by Kaoriasada** [comment]
I'll be on by then.  See you tonight.

**Comment by Kaoriasada** [comment]
Each time I've logged on, neither of you were online.  @whitemistita  @synafey I need a legit time.

**Comment by Kaoriasada** [comment]
Third day of not showing up.  If you're not around today, I'm closing the ticket.

**Comment by Kaoriasada** [comment]
Items were given to @whitemistita and @synafey closing ticket
